src/app/inicio/Inicio.tsx

```
@@ -1,9 +1,28 @@
-import { Greeting, Texto } from '@impulsogov/design-system';
+import type { SituacaoPorIndicador } from '@/types/inicio';
+import dynamic from 'next/dynamic';
+
+const CardsGrid = dynamic<{
+    situacaoPorIndicador: SituacaoPorIndicador;
+    visao: string;
+}>(() => import('./CardsGrid').then(mod => mod.CardsGrid));
+const Greeting = dynamic<{
+    cargo: string;
+    greeting: string;
+    municipio_uf: string;
```

Biome (1.9.4)

[error] This property name should be in camelCase.



(lint/style/useNamingConvention)